### DISCUSSION TITLE
Can forge ids be used with the fabric mods and vice versa?

### DISCUSSION THREAD

**Discussion by RedEgs** [Sinytra/ForgifiedFabricAPI#113] [Q&A]
I've installed a mining dimension mod and it doesn't seem to spawn the ores from mythical metals, but after looking into the config for the mining dimension mod, it seems that mythical metal ids aren't in the config, Is this because fabric ids cant be used with the forge ones or is this simply a mod issue?

The same seems to be the case for using farmer's delight compatibility mods, they don't expand upon the forge version and it often causes the game to behave weirdly or crash entirely.

**Comment by MSandro** [comment]
It could be because Forge and Fabric use different tags. Many mods require manual configuration effort. For others, you can use Custom DataPacks to customize the tags to improve compatibility.
